go.mod

```diff
@@ -175,6 +179,16 @@ require (
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
 	github.com/apparentlymart/go-textseg/v15 v15.0.0 // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.55.5 // indirect
```

Comment:
why do we still need this? Is it a `go mod tidy` snafu or something actually pulls it as a dependency?

Reply:
This is `go-discover` et al.

Reply:
Also `go-getter`, I opened an issue to see if we can get the AWS SDK updated in that repo.  Would be nice to get this dependency removed.